### PR TITLE
Remove pipenv and some unneeded dependencies

### PR DESCRIPTION
**Pull request by ktrieu**
This PR removes pipenv in favor of pip and virtualenv. It also removes some dependencies from back when this was going to be a server side rendered app.

### CODE REVIEW

**Comment by tyxchen** [comment]
(1) What happened to black?
(2) GitHub is complaining that requirements.txt is a binary file; looks like there's some weird encoding going on.

**Comment by ktrieu** [comment]
It seems pipenv did not output the dev dependencies. 

As for the encoding, it's apparently a known issue, looking into it now.

**Comment by tyxchen** [comment]
Looks like the binary weirdness might be related to it being in UTF-16 (!!) little endian (!!!!).

**Comment by ktrieu** [comment]
Yeah looking into it further, PowerShell's redirection operator fried the encoding. Guess I'll have to do it the old fashioned way.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Django==3.0.6
+djangorestframework==3.11.0
+pathspec==0.8.0
+pyparsing==3.0.0a1
+pytz==2020.1
+regex==2020.5.7
+soupsieve==2.0
+sqlparse==0.3.1
+toml==0.10.0
+typed-ast==1.4.1
```
Comment (vincentmacri) [inline requirements.txt]: Wouldn't it be better to make these version ranges so that a minor bug fix version doesn't cause issues?

Comment (ktrieu) [reply requirements.txt]: I've changed them all to the compatible version specifier.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+appdirs~=1.4.3
+asgiref~=3.2.7
+attrs~=19.3.0
+beautifulsoup4~=4.9.0
+black~=19.10b0
+click~=7.1.2
+Django~=3.0.6
+djangorestframework~=3.11.0
+pathspec~=0.8.0
+pyparsing~=3.0.0a1
```
Comment (tyxchen) [inline requirements.txt]: Can we downgrade this to 2.4.7? I didn't realize I installed an alpha version, apologies.

**Review by tyxchen — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (ktrieu) [reply requirements.txt]: https://github.com/UWmathNEWS/slugline-api/pull/11/commits/d9f4c0561b68b87e354f5921ef3f6b6a6d575d8f

Thy will be done.

**Review by tyxchen — APPROVED** [review]

**Pull request merged**